images/fedora/f37/extra-packages

```diff
@@ -7,6 +7,7 @@ findutils
 flatpak-spawn
 fpaste
 git
+glibc-all-langpacks
```

Comment:
Just adding `glibc-all-langpacks` leaves us with both `glibc-minimal-langpack` and `glibc-all-langpacks` in the image.  It's probably harmless, but is a bit odd.  It will be better to use `dnf swap` as I mentioned before.

Secondly, the `/etc/rpm/macros.image-language-conf` file needs to be removed.  Otherwise, the translations get ripped out of the RPMs.  For example, try:
```
$ rpm --list -q sudo | grep locale
```

None of those files actually exist in the image.

We should reinstall all the packages in the base image whose translations went missing.  This is the same problem that we have with the missing documentation that we try to solve through `missing-docs`.

Ideally we would have some script to help us generate this list.  Otherwise it's too tedious and error-prone.  There's an initial attempt at `gen-docs-list`, which could probably be improved.

Reply:
The first two items were fixed in https://github.com/containers/toolbox/pull/1139

However, we still need to figure out a way to restore all the missing translations that got ripped out from the base image.